src/prefect/context.py

```diff
@@ -629,7 +629,6 @@ def root_settings_context():
     Return the settings context that will exist as the root context for the module.
 
     The profile to use is determined with the following precedence
-    - Command line via 'prefect --profile <name>'
```

Comment:
I think we can add this back since we moved handling for this into the new settings source.

Reply:
good point, restored in [5d324f3](https://github.com/PrefectHQ/prefect/pull/15565/commits/5d324f363d8690d95de4a1cd749e9f93d6074d0e)